### ISSUE TITLE
[mobile] Overview of mobile related CI failures

### ISSUE THREAD

**Issue by matouskozak** [dotnet/runtime#103472]
# Description
This issue provides an overview of dotnet/runtime CI failures related to .NET mobile platforms. This tracking issue is updated manually.

## Platforms
The failures are divided per mobile platform when possible or categorized as "Other Failures".
### iOS/TvOS/MacCatalyst
Issue | Status | Comment
-- | -- | --
https://github.com/dotnet/runtime/issues/101692 | untriaged | TvOS crashes on `System.Formats.Tar` tests
https://github.com/dotnet/runtime/issues/101367 | untriaged | TvOS crashes on `System.Memory` tests
https://github.com/dotnet/runtime/issues/101371 | untriaged | MacCatalyst occassionally crashes on `System.IO.IsolatedStorage` tests

### Android
Issue | Status | Comment
-- | -- | --
https://github.com/dotnet/runtime/issues/100281 | *Disabled* | Linux Bionic crashes across variety of test suits

### Other Failures
Issue | Status | Comment
-- | -- | --
https://github.com/dotnet/runtime/issues/103406 | Resolving | `System.Resources.Extensions.Tests.FormattedObject` crashes across all mobile targets
https://github.com/dotnet/runtime/issues/101370 | Investigating | Interpreter occasionally crashes on `System.Data.Odbc` tests

**Comment by steveisok** [comment]
@matouskozak https://github.com/dotnet/runtime/issues/103406 needs a tweak to not run these tests as BinaryFormatter is no longer supported

**Comment by matouskozak** [comment]
> @matouskozak #103406 needs a tweak to not run these tests as BinaryFormatter is no longer supported

You're right, @adamsitnik is working on that in https://github.com/dotnet/runtime/pull/103471.
